client/src/components/ReviewsBreakdown.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const ReviewsBreakdown = (props) => (
```

Comment:
you can always destructure { reviewStats } so you don't have to deal with the props keyword.